schemas/groups/electrical.json

```diff
@@ -174,6 +178,16 @@
             }
           }]
         }
+      },
+      "chargeTransferred": {
+        "type": "number",
+        "description": "Amount of charge produced(+) or consumed(-) sense last report",
```

Comment:
chargeTransfer and energyTransfer are integral measurements reset to 0 each time their value is reported up.   Would " ... reset to zero after each reporting" be clearer?